content/docs/tutorial/multiple-forms.mdx

```diff
@@ -242,11 +242,13 @@ const IndependentForms = () => {
   return (
     <div>
       <JsonForms
+        data={{}}
```

Comment:
This will hand over a new data instance with each rerender. Therefore the data will always be empty. Please use the standard `useState` style for each data with an empty initial object.